reference/progress.v1.yaml

```diff
@@ -312,316 +162,18 @@ paths:
       description: Get simple progress statements for a specific user
 components:
   schemas:
-    Statement:
-      title: XAPI Statement
+    SimpleProgress:
+      title: SimpleProgress
       type: object
       x-examples:
         example-1:
           id: string
-          actor:
-            $ref: '#/components/schemas/Agent'
-          verb: {}
-          object:
-            $ref: '#/components/schemas/Activity'
-          result:
-            score:
-              scaled: 0
-              raw: 0
-              min: 0
-              max: 0
-            success: true
-            completion: true
-            response: string
-            duration: string
-            extensions: {}
-          context: {}
+          activityId: string
+          studentEckId: string
+          studentId: string
+          status: not-started
           timestamp: string
-          stored: string
-          authority: {}
           version: string
```

Comment:
What info do we put here? Is this just to keep the correct order of events, is timestamp then enough?

Reply:
i have updated it to schemaVersion